.changeset/weak-goats-swim.md

```diff
@@ -0,0 +1,6 @@
+---
+'@vercel/client': major
+'vercel': minor
```

Comment:
I think this should be major as well. We're removing the ability to do a big single file, which could potentially break _something_ for someone.